src/components/Auth/ChangePassword/ChangePassword.react.js

```diff
@@ -207,7 +208,7 @@ export default class ChangePassword extends Component {
                         state.success = true;
                     }
                     else{
-                      msg = response.message+'\n Please Try Again.';
+                      msg = response.message+ '\n Please Try Again.';
```

Comment:
@rishiraj824 Do you think we need to translate "Please Try again" as well. Was this intentionally skipped?

Reply:
Good catch! Ideally this needs to be translated too. @rishiraj824 can you please take a look

Reply:
@sudheesh001 @srungta If you see all these messages are translated as well. Go down to where they are being rendered in the component you will find them being sent to the translate component. https://github.com/fossasia/chat.susi.ai/pull/712/files/f8511116a2e20e73dbb5bfd771f82e5c49a12dc2#diff-e4fbf5b6fceb1d5e5744aa918f9b2d40R321